datashare-app/src/main/java/org/icij/datashare/tasks/TaskView.java

```diff
@@ -39,19 +51,27 @@ public TaskView(MonitorableFutureTask<V> task) {
             progress = task.getProgressRate();
         }
     }
+
+    public TaskView(String name, User user, Map<String, Object> properties) {
+        this(randomUUID().toString(), name, State.INIT, 0, user, null, properties);
+    }
+
     @JsonCreator
-    TaskView(@JsonProperty("name") String name,
+    TaskView(@JsonProperty("id") String id,
```

Comment:
I think we talked about it early (and this might be commented somewhere else) but that would be nice to add as a TODO idea to seperate the `result` from the view.
In my mind the view is supposed to reflect the task state and metadata, it should be quickly and frequently accessed. Result on the other hand might be heavy and long to deserialize, which could prevent quick and frequent access to the view.
That can be address in a later PR, but that's to keep in mind in my opinion.